README.md

```diff
@@ -27,8 +27,35 @@ docker run --name meross-ingestor \
 - ⚠️MEROSS_EMAIL
 - ⚠️MEROSS_PASSWORD
 
-Variables marked with `⚠️` are required
+Variables marked with ⚠️ are required
 
+# Pre-requisites
+The Influx database needs to be created before captured data can be sent to it.
+If you are not sending your data to an existing Influx database, this can be done most easily via the CLI. 
+Execute the `influx` command to get an interactive prompt
+
+```
+root@72ee790882b5:/# influx
+Connected to http://localhost:8086 version 1.8.4
+InfluxDB shell version: 1.8.4
+>
+```
+
+Verify you have admin privs on the Influx server
+
+```
+> show users
+user admin
+---- -----
+root true
+```
+
+Create the database. You can change parameters if you wish, or accept the defaults by simply entering
+```
+> create database meross
+```
+where meross can be replaced with the name of the database you wish to create to capture your data. 
+Enter this name as the database in your `-e INFLUX_DATABASE=database` docker instantiation.
```

Comment:
Added commentary around the need for an Influx db to created before using this container successfully. Less experienced users (including myself up to this point) may be used to containers creating required/default databases if they don't already exist.